GlobalCache.php

```diff
@@ -17,7 +17,8 @@
 // This should ideally be patched upstream, converting the hook used
 // to SidebarBeforeOutput rather than SkinBuildSidebar, which is
 // more appropriate for this extension.
-if ( !$wi->config->get( 'wmgUseGroupsSidebar', $wi->dbname ) ) {
+// Disabling Sidebarcahce per wiki as a temporary solution (see T8732)
+if ( !$wi->config->get( 'wmgUseGroupsSidebar', $wi->dbname ) || $wgDBname === 'solarawiki' ) {
```

Comment:
This should be and not or

Reply:
Why and? Wouldn't that mean that only solarawiki will have it enabled? Since none of the other wikis would meet the requirement of having GroupsSidebar AND being 'solarawiki'.

Reply:
No it should be or

Reply:
Wait it should be and !==

Reply:
Yeah I missed that it was === too.

It should be and not

Reply:
```suggestion
if ( !$wi->config->get( 'wmgUseGroupsSidebar', $wi->dbname ) && $wgDBname !== 'solarawiki' ) {
```